pages/birthdays/index.js

```diff
@@ -0,0 +1,224 @@
+  display: flex;
+  flex-direction: column;
+  padding: 0rem;
+`;
+
+const StyledSingleFriend = styled.article`
+  display: flex;
+  flex-direction: row;
+  background-color: white;
+  border-radius: 10px;
+  justify-content: center;
+  width: 20rem;
+  min-width: 0;
+  margin: 0.5rem;
+  padding: 0rem;
+`;
+
+const StyledFriend = styled.article`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const StyledFriendTextBox = styled.article`
+  display: flex;
+  flex-direction: column;
+  padding: 0rem 1rem 0rem 1rem;
+`;
+
+const StyledNickname = styled.h2`
+  color: black;
+  text-decoration: none;
+  font-size: 1.3rem;
+  margin: 0px;
+`;
+
+const StyledBirthday = styled.p`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: black;
+  text-decoration: none;
+  font-size: 0.8rem;
+  margin: 5px 05px 0px 5px;
+`;
+
+const StyledButtonWrapper = styled.article`
+  display: flex;
+  justify-content: right;
+`;
+
+const StyledProfilePicture = styled.article`
+  margin: 5px;
+`;
+
+function formatDate(date) {
+  const day = String(date.getDate()).padStart(2, "0");
+  const month = String(date.getMonth() + 1).padStart(2, "0");
+  const year = date.getFullYear();
+
+  return `${day}.${month}.${year}`;
+}
+
+export default function BirthdayList({ state }) {
+  const calculateDaysUntilBirthday = (birthday) => {
+    const today = new Date();
+    const nextBirthday = new Date(
+      today.getFullYear(),
+      birthday.getMonth(),
+      birthday.getDate()
+    );
+    if (nextBirthday < today) {
+      nextBirthday.setFullYear(nextBirthday.getFullYear() + 1);
+    }
+    const diffTime = Math.abs(nextBirthday - today);
+    const diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+    return diffDays;
+  };
+
+  const [friends, setFriends] = useState(
+    Object.entries(state)
+      .filter(([id, friend]) => friend.birthday)
+      .map(([id, friend]) => ({
+        id,
+        friend,
+        daysUntilBirthday: calculateDaysUntilBirthday(
+          new Date(friend.birthday)
+        ),
+      }))
+      .sort((a, b) => a.daysUntilBirthday - b.daysUntilBirthday)
+  );
+
+  const [visibleFriends, setVisibleFriends] = useState(4);
+
+  const showMoreFriends = () => {
+    setVisibleFriends((prevVisibleFriends) => prevVisibleFriends + 4);
+  };
+
+  const showLessFriends = () => {
+    setVisibleFriends((prevVisibleFriends) => prevVisibleFriends - 4);
+  };
+
+  return (
+    <>
+      <Header />
+      <StyledDiv>
+        <StyledIndex>
+          <StyledBirthdayList>
+            <StyledTitleText>Upcoming birthdays</StyledTitleText>
+
+            <StyledBirthdayBox>
+              {friends
+                .slice(0, visibleFriends)
+                .map(({ id, friend, daysUntilBirthday }) => {
+                  const birthdayDate = new Date(friend.birthday);
+                  const formattedBirthday = formatDate(birthdayDate);
+                  return (
+                    <div key={id}>
```

Comment:
I am not sure if you really need the div here. 
you are wrapping a wrapper :) 
I think this could be avoided in your next PR ;)